AndroidIDPTemplate/app/build.gradle.kts

```diff
@@ -0,0 +1,49 @@
+
+plugins {
+    android
+    `kotlin-android`
+}
+
+dependencies {
+    implementation("com.salesforce.mobilesdk:MobileSync")
+}
+
+android {
+    namespace = "com.salesforce.samples.salesforceandroididptemplateapp"
+
+    compileSdk = 33
+
+    defaultConfig {
+        targetSdk = 33
+        minSdk = 24
+    }
+
+    buildTypes {
+      debug {
+         enableAndroidTestCoverage = true
+      }
+   }
+
+  sourceSets {
+
+        getByName("main") {
```

Comment:
NIT indentation

Reply:
I just checked that his is Android Studio's default.  Did you mean the extra newline above this?

Reply:
Never mind, I thought it looked off in the diff, but it isn't.